### PR TITLE
382: Internals in the Implementation Guide

### PR DESCRIPTION
**Pull request by halprin**
# Internals in the Implementation Guide

Add a starting point for our internal section of the IG.  I think this could be fleshed out in the future.

@jorg3lopez, you mentioned about adding additional documentation around interacting with our API.  This file could be where that documentation resides.

## Issue

#382.

## Checklist

- [n/a] I have added tests to cover my changes
- [n/a] I have added logging where useful (with appropriate log level)
- [n/a] I have added JavaDocs where required
- [x] I have updated the documentation accordingly

**Note**: You may remove items that are not applicable

### CODE REVIEW

File: ig/input/pagecontent/internals.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
 ### Overview
 
-This is where we talk about GitHub and our internal interfaces.
+The ReportStream Intermediary is microservice behind the larger ReportStream system.  This page documents some of the internals behind the microservice.
```
Comment (basiliskus) [inline ig/input/pagecontent/internals.md:3]: Is an `a` missing?: `is microservice` => `is **a** microservice`

Comment (halprin) [reply ig/input/pagecontent/internals.md:3]: Good catch.  English hard.

Comment (halprin) [reply ig/input/pagecontent/internals.md]: Fixed.

**Review by basiliskus — APPROVED** [review]

**Comment by sfradkin** [comment]
I know this is already merged, but is the OpenAPI page accessible on the deployed application?  If so, we should add that link to the documentation as well. @halprin

**Comment by halprin** [comment]
> I know this is already merged, but is the OpenAPI page accessible on the deployed application? If so, we should add that link to the documentation as well. @halprin

The Swagger UI?  No, it is not.  We only have the raw OpenAPI YAML exists.

**Pull request merged**